### PR TITLE
salar / Add download mt5 btn

### PR DESCRIPTION
**Pull request by salar-fs**

### CODE REVIEW

File: src/templates/static/platforms.jsx [hunk 1]
```diff
@@ -15,29 +15,38 @@ const Platforms = ({
...
-            <strong>{description}</strong>
-            <p>{text}</p>
-            <div className='gr-row'>
-                <div className='gr-12'>
-                    { url &&
-                        <a className='button-secondary' download={download || undefined} href={url} target={target || undefined} rel={/http/.test(url) ? 'noopener noreferrer' : undefined}><span>{button_text}</span></a>
-                    }
+    buttons,
+}) => {
+    const btnStyle = { display: 'inline-block', marginRight: '5px' };
```
Comment (negarn) [inline src/templates/static/platforms.jsx]:
```suggestion
    const btn_style = { display: 'inline-block', marginRight: '5px' };
```

Comment (negarn) [reply src/templates/static/platforms.jsx]: it is only used once. why are we declaring it outside? it's changing the indentation of the whole block

File: src/templates/static/platforms.jsx [hunk 2]
```diff
@@ -15,29 +15,38 @@ const Platforms = ({
...
+                        {buttons && buttons.map(button =>
+                            <a
+                                key={button.text}
+                                className='button-secondary'
+                                style={btnStyle}
+                                download={button.download || undefined}
+                                href={button.url}
+                                target={button.target || undefined}
+                                rel={/http/.test(button.url) ? 'noopener noreferrer' : undefined}
+                            ><span>{button.text}</span>
```
Comment (negarn) [inline src/templates/static/platforms.jsx]:
```suggestion
                            >
                                <span>{button.text}</span>
```

File: src/templates/static/platforms.jsx [hunk 3]
```diff
@@ -218,26 +228,28 @@ const Platform = () => (
                                 header={it.L('MetaTrader 5')}
                                 description={it.L('Advanced multi-asset trading platform')}
                                 text={it.L('Trade Forex, CFDs, and binary options with a powerful platform recognised as the global standard.')}
-                                url={it.url_for('user/metatrader')}
-                                button_text={it.L('Access MT5 dashboard')}
+                                buttons={[
+                                    { text: it.L('Access MT5 dashboard'), url: it.url_for('user/metatrader') },
+                                    { text: it.L('Download MT5'), url: 'https://www.binary.com/en/metatrader/download.html' },
```
Comment (negarn) [inline src/templates/static/platforms.jsx]: please don't hardcode this link. use the same `it.url_for` format.

File: src/templates/static/platforms.jsx (hunk 1 above)
Comment (salar-fs) [reply src/templates/static/platforms.jsx]: about indentation, any suggestion?

Comment (negarn) [reply src/templates/static/platforms.jsx]: I mean if we don't make a variable for it, we won't need to add `return (` so the indentations won't change

Comment (salar-fs) [reply src/templates/static/platforms.jsx]: ah OK :) I thought you're talking about the styles and their effect on the buttons.

**Pull request merged**